pyi.py

```diff
@@ -292,7 +292,6 @@ def _check_subscript_version_check(self, node: ast.Compare) -> None:
                 # Python 3.9 flattens the AST and removes Index, so simulate that here
                 slice_num = slc if isinstance(slc, ast.Num) else slc.value
                 # anything other than the integer 0 doesn't make much sense
-                # (things that are in 2.7 and 3.7 but not 3.6?)
                 if isinstance(slice_num, ast.Num) and slice_num.n == 0:
```

Comment:
This code has a potential bug, similar to #60, because `0.0 == 0`. We should check if that can be a problem.